### ISSUE TITLE
How to evaluate pretrained model on specific subset of MPII

### ISSUE THREAD

**Issue by williamrodz** [Microsoft/human-pose-estimation.pytorch#65]
I want like to evaluate the accuracy of the pretrained model with a subset of images of MPII (let's call these images subset A). I have the annotations of subset A in a json file.

I'm a bit confused on how to specifically evaluate the accuracy of the pretrained model on subset A . Where should I place the image files and the annotation json of the subset I want to evaluate?

Right now, I have the following file structure inside ${POSE_ROOT}/data in which I have added the 'subsetA' folder with its corresponding images and annotations (in subsetA/annot/ there only exists a valid.json file the annotations).

data
└── subsetA
│   ├── annot
 ------------valid.json
│   └── images
------------(subsetA images are here)
└── mpii
    ├── annot
    └── images

Specific instructions would be very helpful and greatly appreciated!
